lib/getPackageReportData.js

```diff
@@ -53,7 +53,7 @@ function getPackageReportData(packageEntry, callback) {
 
 		var author = json.author
 		if (_.isObject(author)) {
-			author = author.name;
+			author = author.name || author.email || ""
```

Comment:
why do you add ` || author.email` as `author.email` is handled in the next line already?
Wouldn't the following line not be enough?
```suggestion
			author = author.name || ""
```